src/components/grid/grid.js

```diff
@@ -2,6 +2,7 @@ import React from 'react';
 import classNames from 'classnames';
 import GridIcon from './grid_icon';
 import GridLabel from './grid_label';
+import { Link } from 'react-router';
```

Comment:
@haxpor Thanks for the PR and work to the lint!

I believe to use Link in this case will require dependency from react-router and not everyone use react-router as router even though most of the single page application do. 

Can you modify it to something like [Cell](https://github.com/weui/react-weui/blob/master/src/components/cell/cell.js) that people will be able to pass into their component?